src/Loader.php

```diff
@@ -774,18 +774,14 @@ private static function get_embed_breadcrumbs() {
 	 *
 	 * @param array $section Section to create breadcrumb from.
 	 */
-	private static function output_breadcrumbs( $section ) {
+	private static function output_heading( $section ) {
 		if ( ! static::user_can_analytics() ) {
 			return;
 		}
 		?>
-		<span>
-		<?php if ( is_array( $section ) ) : ?>
-			<a href="<?php echo esc_url( admin_url( $section[0] ) ); ?>"><?php echo esc_html( $section[1] ); ?></a>
-		<?php else : ?>
+		<p class="is-embed">
```

Comment:
This wrapping `p` tag can be removed with the above change.

Reply:
Oh, I like this much better. Committing this and related revisions now :+1: